server/controllers/cards.js

```diff
@@ -19,6 +19,7 @@ module.exports.create = (req, res, company) => {
     notes: req.body.job.notes,
     company_id: company.id,
     user_id: req.user.id,
+    current_status: req.body.status.status,
```

Comment:
Is it supposed to be status.status?

Reply:
Yes, see POST request data coming from the client: 
axios.post('/card', {
      job: {
        title: this.state.title,
        company: this.state.company,
        notes: this.state.notes,
        url: this.state.url,
        recruiter_name: this.state.recruiter_name,
        recruiter_email: this.state.recruiter_email
      },
      status: {
        date: this.state.date,
        status: this.state.status
      }
    })